baremetal/suite_test.go

```diff
@@ -51,8 +51,12 @@ const (
 	clusterName       = "testCluster"
 	metal3ClusterName = "testmetal3Cluster"
 	namespaceName     = "testNameSpace"
+	m3muid            = "a7241a39-4730-44c4-9d81-e70f27a4ce89"
```

Comment:
should not this be:
`m3muid            = "a2e6ab49-9845-4c48-9e49-c74be387f57c"`  based on what I am understanding here:
```
func BuildProviderIDToNodes(bmhuid string, m3muid string) string {

	parts := strings.Split(m3muid, "-")
	firstBlock := parts[0]
	provideruid := fmt.Sprintf("%s_%s", bmhuid, firstBlock)
	return provideruid
}
```
since `var provideruid = "22222222-9845-4c48-9e49-c74be387f57c_a2e6ab49" ` is structured this way

Reply:
For the m3muid, only the first block of the uuid is needed. 
To make it clearer, I have updated the values.